lib/DhallToCabal.hs

```diff
@@ -46,13 +46,9 @@ import Data.Maybe ( fromMaybe )
 import Data.Monoid ( (<>) )
 import Formatting.Buildable ( Buildable(..) )
 
-import qualified Data.ByteString.Lazy as LazyByteString
```

Comment:
Can we drop a whole `bytestring` dependency?

Reply:
Sadly no. Cabal 2.2's parser consumes bytestrings.

Reply:
Not the end of the world, it's a base package after all.